services/web-app/components/article-card/article-card.module.scss

```
@@ -62,6 +62,10 @@
   @include type.type-style('label-01');
```

```suggestion
  @include type.type-style('label-01');
```
```suggestion
```

We should be able to remove this, since we're setting it on the paragraph.